lstchain/calib/camera/utils.py

```diff
@@ -0,0 +1,73 @@
+import tables
+import numpy as np
+
+from lstchain.io.io import dl1_params_tel_mon_ped_key, dl1_params_tel_mon_cal_key
+
+
+def get_bias_and_std(dl1_file):
+    """
+    Function to extract bias and std of pedestal from interleaved events from dl1 file.
+    Parameters
+    ----------
+    input_filename: str
+        path to dl1 file
+    Returns
+    -------
+    bias, std: np.ndarray, np.ndarray
+        bias and std in p.e.
+    """
+    try:
+        f = tables.open_file(dl1_file)
+    except Exception:
+        raise RuntimeError("No found dl1 file")
+    ped = f.root[dl1_params_tel_mon_ped_key]
+    ped_charge_mean = np.array(ped.cols.charge_mean)
+    ped_charge_std = np.array(ped.cols.charge_std)
+    calib = f.root[dl1_params_tel_mon_cal_key]
+    dc_to_pe = np.array(calib.cols.dc_to_pe)
+    ped_charge_mean_pe = ped_charge_mean * dc_to_pe
+    ped_charge_std_pe = ped_charge_std * dc_to_pe
+    f.close()
```

Comment:
please use a context manager for opening and closing files:

```
with tables.open_file(dl1_file) as f:
    ...
```

Reply:
Done